keyboards/gentleman65/gentleman65.c

```diff
@@ -0,0 +1,18 @@
+  
+/* Copyright 2021 Omar Afzal
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "gentleman65.h"
```

Comment:
This sets the behavior by default, and allows for user keymaps to over-ride the behavior. 

Also, this allows the encoders to work for the QMK Configurator. 
As well is updated to use the proper callbacks for the encoder functions (they changed to booleans recently)

```suggestion
#include "gentleman65.h"

bool encoder_update_kb(uint8_t index, bool clockwise) {
    if (!encoder_update_user(index, clockwise)) { return false; }
    if (index == 0) { /* First encoder */
        if (clockwise) {
            tap_code_delay(KC_AUDIO_VOL_UP, 10);
        } else {
            tap_code_delay(KC_AUDIO_VOL_DOWN, 10);
        }
    }
    return true;
}
```